youtube_dl/extractor/wrzuta.py

```diff
@@ -80,3 +80,135 @@ def _real_extract(self, url):
+          second: downloaded on demand by ajax call and rendered
+          using the ajax call response)
+        * in case size of extracted entries not reached total number of entries
+          use the ajax call to collect the remaining entries
+    """
+
+    IE_NAME = 'wrzuta.pl:playlist'
+
+    _VALID_URL = r'https?://(?P<uploader>[0-9a-zA-Z]+)\.wrzuta\.pl/playlista/' \
+                 '(?P<id>[0-9a-zA-Z]+)/.*'
+
+    _TESTS = [{
+        'url': 'http://miromak71.wrzuta.pl/playlista/7XfO4vE84iR/moja_muza',
+        'playlist_mincount': 14,
+        'info_dict': {
+            'id': '7XfO4vE84iR',
+            'title': 'Moja muza',
+        },
+    }, {
+        'url': 'http://noname925.wrzuta.pl/playlista/0uSEHVfbJl2/top_electro_od_noname925',
+        'playlist_mincount': 34,
+        'info_dict': {
+            'id': '0uSEHVfbJl2',
+            'title': 'TOP ELECTRO od noname925',
+        },
+    }, {
+        'url': 'http://panczarek.wrzuta.pl/playlista/52K3cZacAe3/playlista_1',
+        'playlist_mincount': 31,
+        'info_dict': {
+            'id': '52K3cZacAe3',
+            'title': 'playlista #1',
+        },
+    }, {
+        'url': 'http://heroesf70.wrzuta.pl/playlista/6Nj3wQHx756/lipiec_-_lato_2015_muzyka_swiata',
+        'playlist_mincount': 144,
+        'info_dict': {
+            'id': '6Nj3wQHx756',
+            'title': 'Lipiec - Lato 2015 Muzyka Świata',
+        },
+    }, {
+        'url': 'http://andziaoox.wrzuta.pl/playlista/1TOgrSX1hFR/andziaoox_moja_playlista_1',
+        'playlist_mincount': 29,
+        'info_dict': {
+            'id': '1TOgrSX1hFR',
+            'title': 'andziaoox / moja playlista #1',
+        },
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        extracted_id = mobj.group('id')
+        uploader = mobj.group('uploader')
+
+        return self._extracted_playlist_content(
+            url=url,
+            uploader=uploader,
+            playlist_id=extracted_id,
+        )
+
+    def _extracted_playlist_title(self, webpage):
+        return self._html_search_meta(
+            name='og:title',
+            html=webpage,
+            display_name='playlist title'
+        ).replace('Playlista: ', '', 1)
+
+    def _extracted_playlist_content(self, url, uploader, playlist_id):
+        entries = []
+
+        webpage = self._download_webpage(
+            url_or_request=url,
+            video_id=playlist_id,
+        )
+        playlist_size = self._extracted_playlist_size(webpage)
+        playlist_title = self._extracted_playlist_title(webpage)
+
+        there_are_entries_for_extraction = playlist_size > 0
+        if there_are_entries_for_extraction:
+            entries = self._extracted_entries_from_webpage(webpage)
+
+            number_of_extracted_entries_still_lower_than_list_size = playlist_size > len(entries)
+            if number_of_extracted_entries_still_lower_than_list_size:
```

Comment:
It is, but for me was too terse. Instead of putting a comment, I extracted the variable. I'll revert this.